Lib/test/regrtest.py

```diff
@@ -1369,11 +1369,14 @@ def skip_conditional_support(test_module,module_name):
 
     'java.nt':     # Expected to fail on Windows
         """
-        test_mailbox           # fails miserably and ruins other tests
-        # test_os_jy             # Locale tests fail on Cygwin (but not Windows)
-        # test_popen             # Passes, but see http://bugs.python.org/issue1559298
-        test_select_new        # Hangs (Windows), though ok run singly
-        test_urllib2           # file not on local host (likely Windows only)
+        test_mailbox            # fails miserably and ruins other tests
+        # test_popen            # Passes, but see http://bugs.python.org/issue1559298
+        test_select_new         # Hangs (Windows), though ok run singly
+        test_urllib2            # file not on local host (likely Windows only)
+        test_strptime           # Fails on JDK 9+ due to locale strings change
```

Comment:
So, I agree, but I am a bit confused, as this is what I tried to do here. `test_select_new` and `test_urllib2` are excluded as previous, no change (diff artifact). `test_strptime_locale.py` is added to extend `test_strptime` and ensure those tests run, even though `test_strptime` would fail in its CPython form.  Similarly with `test_calendar_locale.py`. Maybe I'm missing something?

Reply:
Having just reviewed #143 I recognise the pattern now. `test_strptime` is disable not because it fails massively, but so that your revised one may run a slightly adjusted version It wasn't as clear here what is going on as in #143.

I still recommend the `_jy` convention.

Reply:
Sure, will continue using the `_jy` suffix rather than making locale special.

Reply:
New patch consistently uses _jy suffix on overriding tests